content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -90,7 +90,7 @@ <h2 style="color: #3771e3;">Deploying your first app on Kubernetes</h2>
 
         <div class="row">
             <div class="col-md-8">
-                <p>For our first Deployment, we'll use a <a href="https://nodejs.org">Node.js</a> application packaged in a Docker container. The source code and the Dockerfile are available in the <a href="https://github.com/kubernetes/website/tree/master/docs/tutorials/kubernetes-basics">GitHub repository</a> for the Kubernetes Basics.</p>
+                <p>For our first Deployment, we'll use a <a href="https://nodejs.org">Node.js</a> application packaged in a Docker container. The source code and the Dockerfile are available in the <a href="https://github.com/kubernetes/website/tree/master/content/en/docs/tutorials/kubernetes-basics">GitHub repository</a> for the Kubernetes Basics.</p>
```

Comment:
Unfortunately, this isn't quite the right location. The files in question are one level up in the repo, at:
https://github.com/kubernetes/website/tree/master/content/en/docs/tutorials